Makefile

```diff
@@ -15,44 +15,45 @@
 ## Open Match Make Help
 ## ====================
 ##
-## Create a GKE Cluster (requires gcloud installed and initialized, https://cloud.google.com/sdk/docs/quickstarts)
+## # Create a GKE Cluster (requires gcloud installed and initialized, https://cloud.google.com/sdk/docs/quickstarts)
```

Comment:
This is (in effect) hotfixing a change to the makefile from the master branch... why?  Did you start working from the wrong branch?  Does this need to be included?

Reply:
Started working from the wrong branch. I will close this PR and submit a new PR once I've started from the correct branch.